Src/libCZI/CziParse.cpp

```diff
@@ -573,6 +573,21 @@ using namespace libCZI;
         {
             libCZI::DimensionIndex dim = CCZIParse::DimensionCharToDimensionIndex(subBlkDirDV->DimensionEntries[i].Dimension, 4);
             entry.coordinate.Set(dim, subBlkDirDV->DimensionEntries[i].Start);
+
+            if(options.GetPhysicalDimensionOtherThanMMustHaveSizeOne())
+            { 
+                auto physicalSize = subBlkDirDV->DimensionEntries[i].StoredSize;
+                if(physicalSize != 1)
+                {
+                    stringstream string_stream;
+                    string_stream 
+                            << "Size for physical dimension '" << Utils::DimensionToChar(dim)
```

Comment:
I'd rather suggest: Physical size for dimension ....

Reply:
Thank you for the catch. Done.